editors/vscode/static/walkthroughs/welcome/get_help.md

```diff
@@ -0,0 +1,10 @@
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
+The Slint has excellent [documentation](https://slint.dev/docs) available online.
+
+You are also free to reach out to the Slint open source community:
+
+-   [Chat with us](https://chat.slint.dev/) on mattermost
+-   [Discuss](https://github.com/slint-ui/slint/discussions) with us on Github
+-   Contact us on [twitter](https://twitter.com/slint_ui) or [mastodon](https://fosstodon.org/@slint)
+-   [Report any issue](https://github.com/slint-ui/slint/issues) you run into on Github
```

Comment:
```suggestion
-   [Report a bug](https://github.com/slint-ui/slint/issues)  on Github
```

(I want to avoid people filling "support issue" (aka questions) on the bug tracker.  I prefer they use the discuss option